addons/web_tour/static/src/js/tip.js

```diff
@@ -365,11 +371,13 @@ var Tip = Widget.extend({
             mbTop -= (this.content_height - this.init_height);
         }
 
+        let contentHeight = this.viewPortState === "in" ? this.content_height : this.scrollContentHeight;
+
         this.$el.toggleClass("inverse", overflow);
         this.$el.removeClass("o_animated").addClass("active");
         this.$el.css({
             width: this.content_width,
-            height: this.content_height,
+            height: contentHeight,
```

Comment:
The width may change too I think? In any case, use a `scrollContentWidth` for safety and consistency.

Reply:
I don't think the width can change or should change, the width is hard coded in the init of the tip.